### DISCUSSION TITLE
SSR in Micro Frontends?

### DISCUSSION THREAD

**Discussion by bernardop** [web-infra-dev/modern.js#4933] [General]
Are there any concrete plans to support SSR in Micro Frontends? Looking at #4741 I see "Better support for Module Federation" and "Better support for Micro Frontend". Is SSR included in those support improvements? Any ETA?

**Comment by zhoushaw** [comment]
Thank you for your attention. We are currently supporting related functions. If there are any latest developments, we will notify you in this discussion

**Reply by zhoushaw** [reply]
We will release our relevant function iteration plan in the near future

**Comment by zhoushaw** [comment]
Can provide more information and our thoughts @ScriptedAlchemy

**Comment by ScriptedAlchemy** [comment]
Hey!
Okay so we have this working internally. 
There are some other important aspects under way, like creating an ehenaced module federation plugin.
Ill see if i can find a little time to see what happens if one were to just apply the node federation plugin directly. 
But, we do have SSR working internally, so this is something you could expect to see in the future iteration
